### ISSUE TITLE
About the training details for the pre-trained model

### ISSUE THREAD

**Issue by yangfengt123** [RanhaoKang/PWC-Net_pytorch#11]
Hi RanhaoKang, 

Thanks for sharing the code. I love your work and wonder if it is convenient for you to inform more detailed settings of your pre-trained model. 
1. I found there are several options available in the code, such as "input normal", "batch normalization",  and "residual", which actually were not implemented in the official code. Did you enable all of them during the training, or just partially, or even did not use them? 

2. I also noticed the pre-trained model are named as "SintelFinal-200K-noBN_SintelFinal-148K-BN.pkl". Does it mean you train the model by the first train on the Sintel Final data with 200K steps without batch normalization and then fine-tune it one the same dataset with BN for another  148K? Have you ever tried to follow the training schedule stated on the paper (FlyingChairs -> FlyingThings -> Sintel/ KITTI) and evaluated the accuracy? 

I believe there are several interesting ideas with your implementations and plan to reproduce your training results. Thanks!

**Comment by xianshunw** [comment]
Hi, did you train the model on FlyingChairs? what's your final EPE error?

**Comment by milts10** [comment]
hello @yangfengt123 did you reproduce the training resutls after all ??

**Comment by xianshunw** [comment]
@milts10 I think the loss in this repo is not exactly the same as the original paper. In this case, this repo performance worse than the original paper slightly.
